libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1064,14 +1064,14 @@ bool AP_AHRS_DCM::get_location(Location &loc) const
     loc.lng = _last_lng;
     const auto &baro = AP::baro();
     const auto &gps = AP::gps();
+    int32_t alt_cm;
     if (_gps_use == GPSUse::EnableWithHeight &&
         gps.status() >= AP_GPS::GPS_OK_FIX_3D) {
-        loc.alt = gps.location().alt;
+        alt_cm = gps.location().alt;
     } else {
-        loc.alt = baro.get_altitude() * 100 + AP::ahrs().get_home().alt;
+        alt_cm = baro.get_altitude() * 100 + AP::ahrs().get_home().alt;
```

Comment:
I know this is existing code, but I wonder if it should be `baro.get_altitude_AMSL()` The difference is home altitude vs the set field elevation.

Reply:
maybe in another PR, this shouldn't be changing behaviour